UnityProject/Assets/Scripts/Items/LastTouch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using UnityEngine;
+
+namespace Items
+{
+	public class LastTouch : MonoBehaviour
```

Comment:
uhhhm could this maybe just be part of pickupable?

Reply:
The S.O.L.I.D. principal is a principal that says systems should be open to extension and closed to modifications. 

Pickupable should only handle picking up the object. Last Touch should only handle checking who was the last one to touch it. Both components have different responsibilities.

Reply:
recording who last picked up something **IS** part of picking up stuff.